src/prpy/base/robot.py

```diff
@@ -459,7 +459,10 @@ def ExecuteTrajectory(self, traj, defer=False, timeout=None, period=0.01, **kwar
 
         # Verify that the trajectory is timed.
         if traj.GetDuration() <= 0.0:
```

Comment:
This check conflates two types of errors:

1. untimed trajectory that is missing the `deltatime` group
2. timed trajectory with zero duration

These have very different causes, so they should produce different error messages. I am fine with leaving (2) as a warning, for now, until we squash the remaining bugs in the planning pipeline. Can you add a check for (1) that throws an exception?